frontend/src/components/Graph.jsx

```diff
@@ -9,19 +9,19 @@ function Graph({ rawDocs, displayedDocs }) {
 
   useEffect(() => {
     let docs = rawDocs || [];
-    const types = Array.from(new Set(docs
-      .flatMap(d => d[2])
-      .filter(x => x !== undefined)
-      .map(x => x.verb)
-    ));
-    const typeColorScale = scaleOrdinal(types, schemeCategory10);
+    const types = {
+      adapts: 'is adapted from',

```

Comment:
please note that one of the inverse label should be changed:
if `B--adapts-->A`  then `A--is-adapted-as-->B`, and not `from B` since `B` is more recent then `A`.